README.md

```diff
@@ -42,8 +42,31 @@ Further, DoDiscover seeks to help the user feel confident with their assumptions
 
 ## Unite causal discovery and causal representation learning
 
-Causal representation learning is the task of learning high-level latent variables and the causal structure between them from low-level variables observed in data.
-DoDiscover seeks to support causal representation learning algorithms in the context of traditional causal discovery settings.
+DoDiscover is a Python library for causal discovery (causal structure learning).
+Our goal is to provide developers and researchers with guide rails for causal discovery that doesn't require deep knowledge of individual causal discovery algorithms.
+
+## What is the difference between dodiscover and other pywhy packages?
+
+The goal of dodiscover is to flatten the on-ramp to causal discovery algorithms.
+DoWhy provides a consistent API for various causal tasks that typically require a graph structure.
+Similarly, DoDiscover aims to provide a cohesive and user-friendly API to apply causal discovery algorithms for inferring a causal graph from data.
+
+[causal-learn](https://github.com/py-why/causal-learn) is an extensive collection of causal discovery algorithms.
+It continuous to host new cutting-edge algorithms in causal discovery.
+However, these algorithms do not have a unified API.
+Further, the historic focus of causal-learn is increasing the capabilities of discovery algorithms.
+In constrast, dodiscover's focus is on the discovery API and usability.
+
+When possible, dodiscover prefers to provide an API wrapper to discovery algorithms in causal-learn and other libraries.
+Please consider contributing to [causal-learn](https://github.com/py-why/causal-learn) if you plan to implement an algorithm from scratch, then contributing a wrapper in dodiscover.
+
+In the future we plan on trying to integrate the two libraries.
+
+## What is the relationship with pywhy-graphs and py-indep?
```

Comment:
I think the new library is to be called pywhy-stats, if I followed the discussion on discord